hexa/metrics/middlewares.py

```diff
@@ -21,13 +21,14 @@ def middleware(request: HttpRequest) -> HttpResponse:
             and request.user.is_authenticated
             and request.method in ("GET", "POST")
             and not request.META.get("PATH_INFO", "").startswith("/admin")
-            and not request.META.get("PATH_INFO", "").startswith("/metrics")
+            and not request.META.get("HEXA_DO_NOT_TRACK", "false") == "true"
```

Comment:
(I added `do_not_track` for the redirect view, also useful to test the decorator)

Reply:
smaaaaart :)